lib/rules/selector-class-pattern/index.mjs

```diff
@@ -89,15 +89,15 @@ const rule = (primary, secondaryOptions) => {
 					return;
 				}
 
-				const selector = String(classNode);
```

Comment:
`String(classNode).trim()` or `classNode.spaces.before = ''` would fix it too.

Reply:
I think I prefer `classNode.toString().trim()` so that it aligns with other cases similar to this.